src/test/java/com/google/sps/data/friend_map/FriendsMapCreationTest.java

```diff
@@ -48,85 +45,75 @@
-
-  @Before
-  public void setUp() {
-    userAFriends = ImmutableSet.of();
-    userBFriends = ImmutableSet.of();
-    userCFriends = ImmutableSet.of(USER_E_ID);
-    userDFriends = ImmutableSet.of(USER_E_ID);
-    userEFriends = ImmutableSet.of(USER_C_ID, USER_D_ID);
-    userA = new UserNode(USER_A_ID, userAFriends);
-    userB = new UserNode(USER_B_ID, userBFriends);
-    userC = new UserNode(USER_C_ID, userCFriends);
-    userD = new UserNode(USER_D_ID, userDFriends);
-    userE = new UserNode(USER_E_ID, userEFriends);
-  }
 
   /**
-  * Tests if graph is forming correctly when there is only one user with 0 friends.
+  * Tests if map is forming correctly when there is only one user with 0 friends.
   *
-  * Should result in a UserFriendsGraph with a map that has a single key that maps
+  * <p>Should result in a UserFriendsMap that has a single key that maps
   * to an empty friend set.
   */
   @Test
   public void oneUserGraphTest() {
+    userA = new UserNode(USER_A_ID, ImmutableSet.of());
+
     Set<UserNode> oneUserSet = new HashSet<>();
     oneUserSet.add(userA);
     UserFriendsMap oneUserFriendsMap = new UserFriendsMap(oneUserSet);
 
     Map<String, ImmutableSet<String>> expectedMap = new HashMap<>();
-    expectedMap.put(USER_A_ID, emptyStringSet);
+    expectedMap.put(USER_A_ID, ImmutableSet.of());
 
     Assert.assertEquals(oneUserFriendsMap.getFriendMap(), ImmutableMap.copyOf(expectedMap));
   }
 
   /**
-  * Tests if graph is forming correctly when there are two users with 0 friends.
+  * Tests if map is forming correctly when there are two users with 0 friends.
   *
-  * Should result in a UserFriendsGraph with a map that has two keys each of which
+  * <p>Should result in a UserFriendsMap that has two keys each of which
   * map to an empty friend set.
   */
   @Test
   public void twoUsersNoConnectionsGraphTest() {
+    userA = new UserNode(USER_A_ID, ImmutableSet.of());
+    userB = new UserNode(USER_B_ID, ImmutableSet.of());
+
     Set<UserNode> twoUserSet = new HashSet<>();
     twoUserSet.add(userA);
     twoUserSet.add(userB);
     UserFriendsMap twoUserFriendsMap = new UserFriendsMap(twoUserSet);
 
     Map<String, ImmutableSet<String>> expectedMap = new HashMap<>();
-    expectedMap.put(USER_A_ID, emptyStringSet);
-    expectedMap.put(USER_B_ID, emptyStringSet);
+    expectedMap.put(USER_A_ID, ImmutableSet.of());
+    expectedMap.put(USER_B_ID, ImmutableSet.of());
 
     Assert.assertEquals(twoUserFriendsMap.getFriendMap(), ImmutableMap.copyOf(expectedMap));
   }
 
   /**
-  * Tests if graph is forming correctly when there are just 3 users, 2 of which are
+  * Tests if map is forming correctly when there are just 3 users, 2 of which are
   * friends with the third user. There are no connections other than this.
   *
-  * Should result in a UserFriendsGraph with three keys:
+  * <p>Should result in a UserFriendsMap with three keys:
   *    User C should map to a friend set with just User E
```

Comment:
Up to you, but there's no reason you can't use users A, B, C here. Reusing is totally fine.

Reply:
Ok, got it!